<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TsoroYematatu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(249,184,81,255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGraphicsView" name="tabuleiro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="renderHints">
     <set>QPainter::Antialiasing|QPainter::LosslessImageRendering|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="textInput">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#textInput{
	border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);
	border-left: none;
	border-top: none;
	background-color: rgba(249,184,81,255);
	padding-left: 10px;
}

#textInput:hover{
	background-color: rgba(253,194,102,255);
}</string>
    </property>
    <property name="placeholderText">
     <string>Digite aqui sua mensagem!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendText">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>420</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#sendText{
	border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);
	border-left: none;
	border-top: none;
	background-color: rgba(251,161,21,255);
}

#sendText:hover{
	background-color: rgba(249,184,81,255);
}</string>
    </property>
    <property name="text">
     <string>ENVIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pos1">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>314</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pos1{
border: none;
border-radius: 13px;
background-color: rgba(0, 0, 0, 0);
}

#pos1:hover{
border: none;
border-radius: 13px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pos2">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>314</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pos2{
border: none;
border-radius: 13px;
background-color: rgba(0, 0, 0, 0);
}

#pos2:hover{
border: none;
border-radius: 13px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pos3">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>314</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pos3{
border: none;
border-radius: 13px;
background-color: rgba(0, 0, 0, 0);
}

#pos3:hover{
border: none;
border-radius: 13px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pos4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>424</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pos4{
border: none;
border-radius: 13px;
background-color: rgba(0, 0, 0, 0);
}

#pos4:hover{
border: none;
border-radius: 13px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pos5">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>424</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pos5{
border: none;
border-radius: 13px;
background-color: rgba(0, 0, 0, 0);
}

#pos5:hover{
border: none;
border-radius: 13px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pos6">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>424</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pos6{
border: none;
border-radius: 13px;
background-color: rgba(0, 0, 0, 0);
}

#pos6:hover{
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pos0">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>151</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pos0{
border: none;
border-radius: 13px;
background-color: rgba(0, 0, 0, 0);
}

#pos0:hover{
border: none;
border-radius: 13px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="textoServidor">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>BEM-VINDO!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QListWidget" name="chatArea">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>501</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#chatArea{
	border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);
	border-left: none;
	padding-left: 10px;
}</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::CurrentChanged</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="empateBTN">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>550</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#empateBTN{
	border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);
	background-color: rgba(251,161,21,255);
	border-left: none;
}

#empateBTN:hover{
	background-color: rgba(249,184,81,255);
}</string>
    </property>
    <property name="text">
     <string>EMPATE</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>5</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>995</x>
      <y>460</y>
      <width>5</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>595</y>
      <width>501</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>5</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 5px;
    border-style: solid;
    border-color: rgba(141,98,57,255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>textoServidor</zorder>
   <zorder>tabuleiro</zorder>
   <zorder>sendText</zorder>
   <zorder>pos1</zorder>
   <zorder>pos2</zorder>
   <zorder>pos3</zorder>
   <zorder>pos4</zorder>
   <zorder>pos5</zorder>
   <zorder>pos6</zorder>
   <zorder>pos0</zorder>
   <zorder>empateBTN</zorder>
   <zorder>textInput</zorder>
   <zorder>chatArea</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
